practice-areas/accessibility-how-to.md

```diff
@@ -0,0 +1,37 @@
+# Accessibility: How To
+
+## Resources for learning how to test the accessibility of our applications
+
+It is worth looking over the [resources on testing on Drupal.org](https://www.drupal.org/docs/accessibility/how-to-do-an-accessibility-review). The following resources are a more consolidated list put in order of priority. They are structured from lowest to greatest effort, and addressing issues in steps 1 & 2 will make screen reader and voice control testing easier:
+
+1/ [Accessibility Insights](https://accessibilityinsights.io/) (FastPass)
+
+Microsoft's open source Accessibility Insights offers both the automated testing with axe and guided keyboard only testing within their FastPass. This is roughly equivalent to using WebAim's proprietary [Wave](https://wave.webaim.org/) toolbar with \[Keyboard-only testing\](https://webaim.org/techniques/keyboard/). The main advantage of Accessibility Insights is that it provides assets which make it easier to include in JIRA to replicate the results.
```

Comment:
@mgifford looks like this link `\[Keyboard-only testing\](https://webaim.org/techniques/keyboard/)` is being escaped.